src/index.js

```diff
@@ -54,13 +59,41 @@ const adapter = new BotFrameworkAdapter({
   appPassword: endpointConfig.appPassword || process.env.MicrosoftAppPassword,
 });
 
-// Create the AdaptiveCardsBot.
-const adaptiveCardsBot = new AdaptiveCardsBot();
+// Define the state store for your bot. See https://aka.ms/about-bot-state to learn more about using MemoryStorage.
+// A bot requires a state storage system to persist the dialog and user state between messages.
+const memoryStorage = new MemoryStorage();
+
+// CAUTION: You must ensure your product environment has the NODE_ENV set
+//          to use the Azure Blob storage or Azure Cosmos DB providers.
+// const { BlobStorage } = require('botbuilder-azure');
+// Storage configuration name or ID from .bot file
+// const STORAGE_CONFIGURATION_ID = '<STORAGE-NAME-OR-ID-FROM-BOT-FILE>';
+// // Default container name
+// const DEFAULT_BOT_CONTAINER = '<DEFAULT-CONTAINER>';
+// // Get service configuration
+// const blobStorageConfig = botConfig.findServiceByNameOrId(STORAGE_CONFIGURATION_ID);
+// const blobStorage = new BlobStorage({
+//     containerName: (blobStorageConfig.container || DEFAULT_BOT_CONTAINER),
+//     storageAccountOrConnectionString: blobStorageConfig.connectionString,
+// });
+
+// Create conversation state with in-memory storage provider.
+const conversationState = new ConversationState(memoryStorage);
+const userState = new UserState(memoryStorage);
+
+// Create the main dialog.
+let bot;
+try {
+  bot = new OrbyBot(conversationState, userState, botConfig);
+} catch (err) {
+  console.error(`[botInitializationError]: ${err}`);
+  process.exit();
+}
 
 // Listen for incoming requests.
 server.post('/api/messages', (req, res) => {
   adapter.processActivity(req, res, async context => {
-    await adaptiveCardsBot.onTurn(context);
+    await bot.onTurn(context);
```

Comment:
more comments that are toxic -- and lead people to believe that i am being mean and bad
hate
garbage
boogers
bad bad bad
loser horrible dumb worst loser